File: Back-end/PetStop_API/PetStop_API/Models/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PetStop_API.Models
{
    public class Pessoa
    {
        public int Pessoa_ID { get; set; }
        //public int Pessoa_ID2 { get; set; }
        public string Nome { get; set; }
        public int Idade { get; set; }
        public string Cpf { get; set; }
        public string Telefone { get; set; }
        public DateTime Dt_Nascimento { get; set; }
        public string Email { get; set; }
        
        #nullable enable
        public Endereco Endereco { get; set; }
        public List<Alergia> Alergia { get; set; }
    }
}

File: Front-end/PetStop/src/app/shared/pessoa.ts
export class Pessoa {
  pessoa_ID: number = 0;
  nome: string = '';
  idade: number = 0;
  cpf: string = '';
  telefone: string = '';
  dt_Nascimento: string = '';
  email: string = '';
  confirmarEmail: string = '';
  senha: string = '';
  confirmarSenha: string = '';
  endereco: any = {
    endereco_ID: null,
    rua: null,
    numero: null,
    complemento: null,
    bairro: null,
    cidade: null,
    cep: null,
    estado: null,
    sigla: null,
  };
  /* alergia: any = {
    alergia_ID: null,
    nome: null,
  }; */
}